core/computers.go

```diff
@@ -105,6 +109,30 @@ func GetPercentageOfValue(value *big.Int, percentage float64) *big.Int {
 	return result
 }
 
+// GetIntPercentageOfValue returns the exact percentage of value, that fits into the integer (with loss of division remainder)
```

Comment:
comment does not match the function name

Reply:
done